css-color-5/Overview.bs

```diff
@@ -745,20 +745,20 @@ Unless the type of hue interpolation is ''specified'', both angles need to be co
 One way to do this is <code><i>θ</i> = ((<i>θ</i> % 360) + 360) % 360</code>.
 
 : ''shorter''
-:: Angles are adjusted so that θ₂ - θ₁ ∈ [0, 180). In pseudo-Javascript:
+:: Angles are adjusted so that θ₂ - θ₁ ∈ [-180, 180). In pseudo-Javascript:
 	<pre>
 	if (θ₂ - θ₁ >= 180) {
 		θ₁ += 360;
 	}
-	else if (θ₂ - θ₁ <= -180) {
+	else if (θ₂ - θ₁ < -180) {
 		θ₂ += 360;
 	}
 	</pre>
 
 : ''longer''
-:: Angles are adjusted so that θ₂ - θ₁ ∈ [180, 360). In pseudo-Javascript:
+:: Angles are adjusted so that θ₂ - θ₁ ∈ {(-360, -180], 0, (180, 360)}. In pseudo-Javascript:
```

Comment:
I find this rather hard to read, can we prioritize readability over conciseness?

Reply:
By just changing it to an English sentence rather than set notation?  Or some other way?

Reply:
0 is not needed if we make the sets closed, and we can combine [-360, -180] and [180, 360] by using the absolute value of the difference.

Reply:
I'm still pretty hesitant to write a rule that says that interpolation of *identical* hues goes around the circle -- both because we'd have to choose which way, and because I think it might be problematic if we applied such rules to transitions and animations, which I think may depend in some ways on interpolation of identical values not going through other values.

I'll make the other changes shortly.

Reply:
Makes sense, yeah.